bin/BuildPackages.sh

```diff
@@ -39,7 +49,9 @@ if [ -f gapicon.bmp ]
 fi
 
 # We need any subdirectory, to test if $GAPDIR is right
-SUBDIR=`ls -d */ | head -n 1`
+# dangerous to parse output of ls
+#SUBDIR=`ls -d */ | head -n 1`
+SUBDIR=`find -L ./ -maxdepth 1 -mindepth 1 -type d | sed 's|./||' | head -n 1`
```

Comment:
I realize that you are just adapting existing code here, so I'll accept this. But I actually think it's not a great solution. Better would be to convert any relative paths into absolute ones ... in particular, don't require the user to specify a path "relative from a package directory. Instead, require the user to specify a path which is valid from the place `BuildPackages.sh` is called (i.e. inside some `pkg` directory). Then, if that path is relative, convert it to an absolute path. And then use that. So in particular, we'd not default to the GAPDIR `../..` but rather `..`. 

But again, I am not asking you to make that change now, just pointint this out.